refinery/ui/source/js/file-browser/views/files-tab.html

```diff
@@ -43,7 +43,7 @@
         </span>
 
       </div>
-      <div id="table-filter">
+      <div id="table-filter" class="row">
```

Comment:
row?

Reply:
The top div and the bottom div of that column were different widths because the top had `class="row"` and the bottom didn't ... simply removing that class meant the wrench button would sometimes wrap below and sit to the right of the `table-filter` div.

If there's something that would be better, let us know?